runtime-modules/content/src/lib.rs

```diff
@@ -699,10 +699,27 @@ decl_module! {
                         params.clone(),
                     )?;
                 }
-                // create_dynamic_bag_with_objects with its can* guard ensures that this invocation succeeds
+                // create_dynamic_bag_with_objects with its can* guard ensures that this invocation succee ds
                 Storage::<T>::upload_data_objects(params)?;
-            }
+            } else {
+                if Storage::<T>::ensure_bag_exists(&params.bag_id).is_err() {
+                    Storage::<T>::can_create_dynamic_bag(
+                        &DynamicBagIdType::<T::MemberId, T::ChannelId>::Channel(channel_id),
+                        &Some(deletion_prize.clone()),
+                    )?;
+                }
+
+                //
```

Comment:
We can't have several `mutation safe` marks. The method became overcomplicated with code duplication. Please, refactor it. It also contains a double empty line - please fix it.

Reply:
ok